config/shared.lua

```diff
@@ -2,7 +2,226 @@ return {
+            fmj = 'luxury',
+            furia = 'luxury',
+            gp1 = 'luxury',
+            infernus = 'luxury',
+            italigtb = 'luxury',
+            italigtb2 = 'luxury',
+            krieger = 'luxury',
+            le7b = 'luxury',
+            lm87 = 'luxury',
+            nero = 'luxury',
+            nero2 = 'luxury',
+            omnisegt = 'luxury',
+            osiris = 'luxury',
+            penetrator = 'luxury',
+            pfister811 = 'luxury',
+            prototipo = 'luxury',
+            reaper = 'luxury',
+            s80 = 'luxury',
+            sc1 = 'luxury',
+            sentinel4 = 'luxury',
+            sheava = 'luxury',
+            sm722 = 'luxury',
+            sultanrs = 'luxury',
+            t20 = 'luxury',
+            taipan = 'luxury',
+            tempesta = 'luxury',
+            tenf = 'luxury',
+            tenf2 = 'luxury',
+            torero2 = 'luxury',
+            tezeract = 'luxury',
+            thrax = 'luxury',
+            tigon = 'luxury',
+            turismor = 'luxury',
+            tyrant = 'luxury',
+            tyrus = 'luxury',
+            vacca = 'luxury',
+            vagner = 'luxury',
+            visione = 'luxury',
+            voltic = 'luxury',
+            voltic2 = 'luxury',
+            xa21 = 'luxury',
+            zentorno = 'luxury',
+            zorrusso = 'luxury',
+
+            --- Boats
+            squalo = 'boats',
+            marquis = 'boats',
+            seashark = 'boats',
+            seashark2 = 'boats',
+            seashark3 = 'boats',
+            jetmax = 'boats',
+            tropic = 'boats',
+            tropic2 = 'boats',
+            dinghy = 'boats',
+            dinghy2 = 'boats',
+            dinghy3 = 'boats',
+            dinghy4 = 'boats',
+            suntrap = 'boats',
+            speeder = 'boats',
+            speeder2 = 'boats',
+            longfin = 'boats',
+            toro = 'boats',
+            toro2 = 'boats',
+
+            --- Helicopters
+            buzzard2 = 'air',
+            frogger = 'air',
+            frogger2 = 'air',
+            maverick = 'air',
+            swift = 'air',
+            swift2 = 'air',
+            seasparrow = 'air',
+            seasparrow2 = 'air',
+            seasparrow3 = 'air',
+            supervolito = 'air',
+            supervolito2 = 'air',
+            volatus = 'air',
+            havok = 'air',
+
+            --- Planes
+            duster = 'air',
+            luxor = 'air',
+            luxor2 = 'air',
+            stunt = 'air',
+            mammatus = 'air',
+            velum = 'air',
+            velum2 = 'air',
+            shamal = 'air',
+            vestra = 'air',
+            dodo = 'air',
+            howard = 'air',
+            alphaz1 = 'air',
+            nimbus = 'air',
+            conada = 'air',
+        },
+
+        blocklist = {
+            'police',
+            'police2',
+            'police3',
+            'police4',
+        }
```

Comment:
Rather than a separate block list, one could make the same entry on the model list but set the value to empty table to accomplish the same thing. Do you think this is preferable over a separate blocklist?

Reply:
Both would work but I think from a clarity point of view having it as a separate table is a bit easier from an end user perspective

Reply:
Why do we even need a blacklist? They already add vehicles they want to shops. No point to blacklist vehicles which wont be added.

Reply:
> Why do we even need a blacklist? They already add vehicles they want to shops. No point to blacklist vehicles which wont be added.

the default config adds all cars to pdm unless I read it wrong

Reply:
No youre correct. I forgot we did that